chronograf.go

```diff
@@ -471,6 +485,24 @@ type Databases interface {
 	DropRP(context.Context, string, string) error
 }
 
+// Annotation represents a time-based metadata associated with a source
+type Annotation struct {
```

Comment:
Should the dbrp and measurement name be attributes on the struct here rather than on point?

Reply:
I think db/rp should be on the point because a point "represents" line protocol.

However, I do think we need some way of specifying db/rp for all of the annotation collection.  I had pushed that into the future, but, I was thinking that it could be a setting on the source object similar to the default telegraf db.